jekyll/_cci2/using-docker.md

```diff
@@ -220,6 +220,79 @@ In all cases, cache hits are not guaranteed, but are a bonus convenience when av
 
 In summary, the availability of caching is not something that can be controlled via settings or configuration, but by choosing a popular image, such as [CircleCI convenience images](https://circleci.com/developer/images), you will have more chances of hitting cached layers in the "Spin Up Environment" step.
 
+## Run a job in a container on your machine
```

Comment:
I feel like this should be it's own How To page outside of the main Docker execution environments page